neard/res/mainnet_genesis.json

```diff
@@ -0,0 +1,336 @@
+  "num_block_producer_seats": 100,
+  "num_block_producer_seats_per_shard": [
+    100
+  ],
+  "avg_hidden_validator_seats_per_shard": [
+    0
+  ],
+  "dynamic_resharding": false,
+  "epoch_length": 43200,
+  "gas_limit": 1000000000000000,
+  "min_gas_price": "100000",
+  "block_producer_kickout_threshold": 90,
+  "chunk_producer_kickout_threshold": 60,
+  "gas_price_adjustment_rate": 1,
+  "runtime_config": {
+    "storage_amount_per_byte": "90900000000000000000",
+    "transaction_costs": {
+      "action_receipt_creation_config": {
+        "send_sir": 924119500000,
+        "send_not_sir": 924119500000,
+        "execution": 924119500000
+      },
+      "data_receipt_creation_config": {
+        "base_cost": {
+          "send_sir": 539890689500,
+          "send_not_sir": 539890689500,
+          "execution": 539890689500
+        },
+        "cost_per_byte": {
+          "send_sir": 14234654,
+          "send_not_sir": 14234654,
+          "execution": 14234654
+        }
+      },
+      "action_creation_config": {
+        "create_account_cost": {
+          "send_sir": 0,
+          "send_not_sir": 0,
+          "execution": 0
+        },
+        "deploy_contract_cost": {
+          "send_sir": 513359000000,
+          "send_not_sir": 513359000000,
+          "execution": 513359000000
+        },
+        "deploy_contract_cost_per_byte": {
+          "send_sir": 27106233,
+          "send_not_sir": 27106233,
+          "execution": 27106233
+        },
+        "function_call_cost": {
+          "send_sir": 1367372500000,
+          "send_not_sir": 1367372500000,
+          "execution": 1367372500000
+        },
+        "function_call_cost_per_byte": {
+          "send_sir": 2354953,
+          "send_not_sir": 2354953,
+          "execution": 2354953
+        },
+        "transfer_cost": {
+          "send_sir": 13025000000,
+          "send_not_sir": 13025000000,
+          "execution": 13025000000
+        },
+        "stake_cost": {
+          "send_sir": 0,
+          "send_not_sir": 0,
+          "execution": 0
+        },
+        "add_key_cost": {
+          "full_access_cost": {
+            "send_sir": 0,
+            "send_not_sir": 0,
+            "execution": 0
+          },
+          "function_call_cost": {
+            "send_sir": 0,
+            "send_not_sir": 0,
+            "execution": 0
+          },
+          "function_call_cost_per_byte": {
+            "send_sir": 37538150,
+            "send_not_sir": 37538150,
+            "execution": 37538150
+          }
+        },
+        "delete_key_cost": {
+          "send_sir": 0,
+          "send_not_sir": 0,
+          "execution": 0
+        },
+        "delete_account_cost": {
+          "send_sir": 454830000000,
+          "send_not_sir": 454830000000,
+          "execution": 454830000000
+        }
+      },
+      "storage_usage_config": {
+        "num_bytes_account": 100,
+        "num_extra_bytes_record": 40
+      },
+      "burnt_gas_reward": {
```

Comment:
@bowenwang1996 do we need to update the format after https://github.com/nearprotocol/nearcore/pull/2392 ?